scripts/docker/polkadot/polkadot_builder.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM docker.io/paritytech/ci-linux:production as builder
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="builder" \
+	io.parity.image.description="This is the build stage for Polkadot. Here we create the binary." \
```

Comment:
I don't think the builder images need all those labels. Those images are transient and will never be published unlike the second stage. We probably can cut that down to keep the image more readable.
@TriplEight any take on that?

Reply:
The idea behind it was a policy-as-a-code in future that would check that every Dockerfile has metadata. But good point on readability, makes sense to make an exclusion for the files with the specific `*builder*` in the name for such policy.

Reply:
It's OK to have simple explanation comments instead of the full metadata in such files.